examples/plot_virtual_evoked.py

```diff
@@ -0,0 +1,33 @@
+"""
+=======================
+Remap MEG channel types
+=======================
+
+In this example, MEG data are remapped from one
+channel type to another. This is useful for combining statistics
+for magnetometer and gradiometers. This process can be
+computationally intensive.
+"""
+
+# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+
+# License: BSD (3-clause)
+
+from mne.datasets import sample
+from mne import read_evokeds
+from mne.forward import compute_virtual_evoked
+
+print(__doc__)
+
+data_path = sample.data_path()
+evoked_fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
+
+from_type, to_type, condition = 'grad', 'mag', 'Left Auditory'
+evoked = read_evokeds(evoked_fname, condition=condition, baseline=(-0.2, 0.0))
+
+virtual_evoked = compute_virtual_evoked(evoked, from_type=from_type,
+                                        to_type=to_type)
```

Comment:
ok, we have the signs thanks to brainstorming with @teonlamont and @ybekhti. The grad topomap had the RMS values of two pairs, so for interpolated 'grad'. we take the absolute magnitude (since we don't have any pairs) for better comparison:

![mag_to_grad](https://cloud.githubusercontent.com/assets/3817535/7118017/ce82a328-e1fc-11e4-91c4-fda2b0bc5b90.png)

Fig 1 is interpolated grad and Fig 2 is true grad. 

There is still a weird magnitude bias which needs to be figured out. When going from grad to mag the interpolated mag data is higher in magnitude compared to True mag whereas when going from mag to grad the interpolated grad data is lower in magnitude compared to True grad